docs/prow/presets.md

```diff
@@ -26,3 +26,4 @@ This document contains the list of all Presets available in the [`config.yaml`](
 | **preset-kyma-artifacts-bucket** | It defines the environment variable for the Kyma's artifact bucket.                     |
 | **preset-stability-checker-slack-notifications** | It defines a webhook URL and a client token required for the Slack integration.                 |
 | **preset-sap-slack-bot-token** | It sets the environment variable for the Slack token of the bot account in the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).|
+| **preset-kyma-ondemands** | It defines the environment variable for the Kyma's on-demand artifacts bucket. |
```

Comment:
Do we have a place with documentation per job? Will be nice to explain what on-demand artifacts bucket will contain, and as well what triggers an upload (in that case, PR, and every PR change)?

Reply:
I added such comment to `build-ondemand-kyma-artifacts.sh`. We don't have documentation for every job right now